AllInOne/Scripts/kalman_data_collector.py

```diff
@@ -12,15 +13,16 @@ def add_data(self, values_dict):
                 self.data[key].append(self.filters[key].update(value))
 
     def is_valid(self):
+        logging.info(self.data)
         for elem in list(self.data.values()):
-            if len(elem) < 11:
+            if len(elem) < 5:
```

Comment:
@HovhannesApinyan Please change static Numbers to variables and add comments what types is it variable or why that is need